cordova-react/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",         
+        "jsx": "react",
+        "sourceMap": true,
+        "noImplicitAny": true
+    },
+    "files": [
```

Comment:
i would recommend using `exclude` instead of listing the files. this way adding a new file to the sample would not require adding it here.